### DISCUSSION TITLE
Permanent "Glass" effect

### DISCUSSION THREAD

**Discussion by DirtyJohny** [0x7c13/Notepads#1134] [General]
Hello, thank for your project its awesome app. I interesting about your glass effect. Apps: Teminal, Files App have a prolem with permanent theme glass. How you did that? Can you tell me, how such a result turned out when writing my applications in future? I did not really understand what type to make for discussion, I apologize in advance.  Thanks for the answer.

**Comment by 0x7c13** [comment]
https://github.com/0x7c13/Notepads/blob/master/src/Notepads/Brushes/HostBackdropAcrylicBrush.cs
